File: Lang_C/chap5/7.c
//Problem NO.7
#include <stdio.h>
#include <math.h>
int main()
{
    int sum=0;
    for (int i=0;i<=100;i++){
        sum+=i;
    }
    for (int i=1;i<=50;i++){
        sum+=i*i;
    }
    for (int i=1;i<=10;i++){
        sum+=1/i;
    }
    printf("The sum is %d\n",sum);
    return 0;
}

File: POJ/Leetcode/7.java
//Given a 32-bit signed integer, reverse digits of an integer.
//For the purpose of this problem, assume that your function returns 0
//when the reversed integer overflows.
class Solution {
    public int reverse(int x) {
        StringBuilder strBuilder = new StringBuilder(Integer.toString(Math.abs(x)));
        try {
            String str = strBuilder.reverse().toString();
            return Integer.parseInt(str.replaceFirst("^0+(?!$)", "")) * (x/Math.abs(x));
        } catch (Exception e){
            return 0;
        }
    }
}